nrt/data/IEEE/IEEE14_BlackBoxModels/reference/outputs/logs/dynawo.log

```diff
@@ -22,37 +22,37 @@
 2018-12-19 17:10:35 | DEBUG | call of SolverReInit ie a new symbolic and numerical factorization will be performed
 2018-12-19 17:10:35 | DEBUG | -----------------------------------------------------------------------
 2018-12-19 17:10:35 | DEBUG | number of continuous variables : 408 (display limited to 4 variables)
-2018-12-19 17:10:35 | DEBUG | number of discrete variables : 190
+2018-12-19 17:10:35 | DEBUG | number of discrete variables : 196
```

Comment:
I don't get it. Is it from another development that didn't refresh the log file? 
I don't see any reason for having a change in the number of discrete variables due to Sundials version (at least at first glance)

Reply:
Yes this commit includes a fix that now allows to use the dynawo.log as reference (it was broken before) so some references had to be updated independently from the upgrade of sundials version.